cli/src/program.rs

```diff
@@ -268,6 +270,9 @@ impl ProgramSubCommands for App<'_, '_> {
                                     whichever comes first.",
                                 ),
                         )
+                        .arg(Arg::with_name("use_rpc").long("use-rpc").help(
+                            "Send transactions to the configured RPC instead of validator TPUs",
```

Comment:
Technically, the first and last transactions are always sent via RPC already. Does it make sense to update to say "write transactions" or "all transactions"

Reply:
Ah that's a good point, probably best to say "write transactions" since those are the ones that are different. I'll put in another PR for that